packages/components/src/viewport/aware/index.js

```diff
@@ -37,6 +38,9 @@ export default WrappedComponent => {
         this.state.componentOffset == null ||
         this.state.componentHeight == null
       ) {
+        if (!this._isMounted) {
+          return setTimeout(() => this._onViewportChange(), 50)
```

Comment:
I know this is not very nice. But invoking `_onViewportChange` on `componentDidMount` wouldn't do the trick.

Reply:
I moved this part to `componentDidMount` to provide a safer implementation (https://github.com/netceteragroup/girders-elements/pull/77/commits/6d93c9b7a3d8079f5e229e9dd646fda69284a596). But the `setTimeout` had to stay, otherwise `UIManager.measureLayout` throws an error:
`Attempted to measure layout but offset or dimensions were NaN`.